pgmpy/models/BayesianModel.py

```diff
@@ -410,9 +412,9 @@ def active_trail_nodes(self, start, observed=None):
         >>> student.add_nodes_from(['diff', 'intel', 'grades'])
         >>> student.add_edges_from([('diff', 'grades'), ('intel', 'grades')])
         >>> student.active_trail_nodes('diff')
```

Comment:
@raghavg7796 I'm not in a agreement with allowing strings for single variable. @ankurankan  I think we are doing this in other places also. Was there any particular reason for this ?